src/backend/utils/storage.js

```diff
@@ -120,4 +129,14 @@ module.exports = {
   getPostsCount: () => redis.zcard(postsKey),
 
   getPost: (id) => redis.hgetall(postNamespace.concat(id)),
+
+  removePost: async (id) => {
```

Comment:
Add a comment above this about having to do the same in elastic (Feed does this).

Reply:
added comment, not sure if it is what you wanted